certified-connectors/TOPdesk/apiDefinition.swagger.json

```diff
@@ -0,0 +1,11510 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "A Connector for TOPdesk, which entails connectors for Supporting Files, and Incident Management. All endpoint for creating, processing and closing incidents are available in the connector, aswell as alle endpoints for creating and handling branches, locations, persons and operators.",
+    "version": "1.0",
+    "title": "TOPdesk API V1",
+    "contact": {
+      "name": "TOPdesk",
+      "url": "http://www.topdesk.com",
+      "email": "info@topdesk.com"
+    }
+  },
+  "basePath": "/tas/api",
+  "schemes": [
+    "https"
+  ],
+  "x-ms-connector-metadata":[
+  {
+  "propertyName": "Website",
+  "propertyValue": "https://www.topdesk.com/"
+  },
+  {
+  "propertyName": "Privacy Policy",
+  "propertyValue": "https://www.topdesk.com/en/privacy-policy/"
+  },
+  {
+  "propertyName": "Categories",
+  "propertyValue": "Service;Helpdesk;ITSM;IT Operations;ESM"
+  }
+  ]
```

Comment:
@CJusjong Syntax error - This property should end with a comma. Did you test this connector in your local environment?